src/clr-core/button/button.element.scss

```diff
@@ -1,33 +1,37 @@
 @import './../styles/mixins/mixins';
 
-// TODO: progress spinner CSS deprecated, will be replaced by cwc-progress
+// TODO: progress spinner CSS deprecated, will be replaced by cds-progress
 @import './../../clr-angular/progress/spinner/variables.spinner';
 @import './../../clr-angular/image/icons.clarity';
 @import './../../clr-angular/progress/spinner/spinner.clarity';
 
 :host {
   --box-shadow-color: var(--clr-color-action-500, #{$clr-color-action-500});
   --border-radius: #{$clr_baselineRem_3px};
-  --border-width: #{$clr_baselineRem_1px};
-  --font-size: #{$clr_baselineRem_0_5};
-  --font-weight: #{$clr-font-weight-semibold};
   --border-color: var(--clr-color-action-600, #{$clr-color-action-600});
   --color: var(--clr-color-neutral-0, #{$clr-color-neutral-0});
   --background: var(--clr-color-action-600, #{$clr-color-action-600});
+  display: inline-block;
+}
+
+.private-host {
+  --font-size: #{$clr_baselineRem_0_5};
   --line-height: 1.7rem;
   --letter-spacing: 0.12em;
+  --padding: 0 #{$clr_baselineRem_0_5};
+
   background: var(--background);
   color: var(--color);
   border-color: var(--border-color);
   border-radius: var(--border-radius);
-  border-width: var(--border-width);
+  border-width: $clr_baselineRem_1px;
```

Comment:
Right now we have a `--clr-global-borderwidth`. Do we want to drop that?

Reply:
hmmm for core I would think so since we did not want border sizes to be adjustable to prevent reflow. I am not sure the implications for clr-ui

Reply:
Yeah. The problem is we are still using the SASS variable `$clr-global-borderwidth` in places. We should probably be consistent one way or the other.